usb/device/CMakeLists.txt

```diff
@@ -1,4 +1,23 @@
-add_subdirectory(dev_audio_headset)
-add_subdirectory(dev_hid_composite)
-add_subdirectory(dev_hid_generic_inout)
+# not 100% sure what this does
+add_tinyusb_device_example(audio_test)
+add_tinyusb_device_example(board_test)
+add_tinyusb_device_example(cdc_dual_ports)
+add_tinyusb_device_example(cdc_msc)
+# todo gate with freertos lib
+#add_tinyusb_device_example(cdc_msc_freertos)
+add_tinyusb_device_example(dfu_runtime)
```

Comment:
Is this relevant, given that we recommend flashing via UF2? :shrug:

Reply:
what do you mean?

Reply:
DFU = Device Firmware Upgrade, typically used to write the flash on a microcontroller (which is what we use UF2 for). But I've just checked https://github.com/hathach/tinyusb/blob/master/examples/device/dfu_runtime/src/main.c and see that it says:
> Since this example is minimal, it doesn't actually go into DFU mode but rather change the LED blinking pattern to fast rate as indicator.

So apologies for the irrelevant comment! :laughing: